NDLANO/ndla-frontend
# 2531 vise flytta emner

Fixes: https://github.com/NDLANO/Issues/issues/2531
Depends on: https://github.com/NDLANO/graphql-api/pull/152

How to test:
1. ~Wait for ^ graphql merge `or` pull and run graphql locally~
2. Go to search-page and filter results for topics
3. Click any topic from search
4. You should see something like this if the topic have more than one contexts:
![image](https://user-images.githubusercontent.com/10486712/115032863-17835f00-9eca-11eb-9538-ca3c12ce2107.png)

OR you should be redirected to a valid url.
5. Click on the links and check that you are getting redirected correctly
